tests/manage/test_pvc_disruptive.py

```diff
@@ -13,6 +19,40 @@
 DISRUPTION_OPS = disruption_helpers.Disruptions()
 
 
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Setup and teardown
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
```

Comment:
I suggest to have a create_namespace fixture in tests/fixtures and use from here

Reply:
Done